runner.py

```diff
@@ -0,0 +1,163 @@
+import random
+
+class Game:
+    def __init__(self):
+        self.rooms = {
```

Comment:
It would make sense to make a Room class here and then build the instances of the rooms like so!